backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -136,8 +142,9 @@ private String checkForDuplicateOrg(String organizationName, String state, Strin
       Optional<Organization> duplicateOrg =
           potentialDuplicates.stream().filter(o -> o.getExternalId().startsWith(state)).findFirst();
       if (duplicateOrg.isPresent()) {
-        if (_oktaRepo.fetchAdminUserEmail(duplicateOrg.get()).stream()
-            .anyMatch(Predicate.isEqual(email))) {
+        List<String> adminUserEmails = getOrgAdminUserEmails(duplicateOrg.get());
+
+        if (adminUserEmails.stream().anyMatch(Predicate.isEqual(email))) {
```

Comment:
Do we need to handle or otherwise flag if the org doesn't have any admins? (was thinking about no admin users after [this comment](https://github.com/CDCgov/prime-simplereport/pull/8105/files#r1752626902))

Reply:
I think it's OK in this context because we are trying to check if the to-be-created new org might be a duplicate of another org. If there are no org admins in that duplicate org, it would throw the exception on line 155. Let me know what you think!